docs/cristin.ttl

```diff
@@ -944,6 +1009,62 @@ It is a self reference.""" ;
         entityDescription.reference.publicationContext.title
     """ .
 
+:tidsskriftnr
+    a               :cristinfield ;
+    rdfs:comment    "Number identifying the specific article" ;
```

Comment:
How is this different from artikkelnummer?

Reply:
tidsskriftnr er fremmednøkkel til hvilket tidsskrift publikasjonen er publisert i